<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Execute</class>
 <widget class="QWidget" name="Execute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Execute Session</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="KUrlLabel" name="showSessionsButton">
       <property name="text">
        <string>Observing Sessions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="sessionsListWidget"/>
     </item>
     <item>
      <widget class="KUrlLabel" name="showTargetsButton">
       <property name="text">
        <string>Targets and observations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="targetsTreeView"/>
     </item>
     <item>
      <widget class="KUrlLabel" name="addTargetButton">
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string>Add new object to list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="equipmentButton">
       <property name="text">
        <string>Define Equipment</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="sessionPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KTitleWidget" name="ktitlewidget">
           <property name="text">
            <string>Enter Session Details:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="sessionLocationLabel">
             <property name="text">
              <string>Location:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="sessionLocationButton">
             <property name="text">
              <string>set location</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="sessionCoobserversButton">
             <property name="text">
              <string>Select Coobservers...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="sessionBeginLabel">
             <property name="text">
              <string>Begin:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KDateTimeWidget" name="sessionBeginDateTime"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="sessionWeatherLabel">
             <property name="text">
              <string>Weather:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="sessionEquipmentLabel">
             <property name="text">
              <string>Equipment:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="sessionCommentsLabel">
             <property name="text">
              <string>Comments:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="sessionLanguageLabel">
             <property name="text">
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="KLineEdit" name="sessionLanguageLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="sessionEndLabel">
             <property name="text">
              <string>End:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KDateTimeWidget" name="sessionEndDateTime"/>
           </item>
           <item row="4" column="1">
            <widget class="QPlainTextEdit" name="sessionWeatherTextEdit"/>
           </item>
           <item row="5" column="1">
            <widget class="QPlainTextEdit" name="sessionEquipmentTextEdit"/>
           </item>
           <item row="6" column="1">
            <widget class="QPlainTextEdit" name="sessionCommentsTextEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="targetPage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="KTitleWidget" name="targetTitle">
           <property name="text">
            <string>Target Details</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="targetTabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="targetGeneralTab">
            <attribute name="title">
             <string>General</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="targetNameLabel">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="targetNameLineEdit"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="targetAliasesLabel">
                 <property name="text">
                  <string>Aliases:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QListView" name="targetAliasesListView"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="targetSourceLabel">
                 <property name="text">
                  <string>Source: </string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="targetSourceComboBox">
                 <item>
                  <property name="text">
                   <string>Data source</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Observer</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="targetDatasourceLabel">
                 <property name="text">
                  <string>Data source:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="targetDatasourceLineEdit"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="targetObserverLabel">
                 <property name="text">
                  <string>Observer:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="targetConstellationLabel">
                 <property name="text">
                  <string>Constellation:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="targetConstellationLineEdit"/>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="targetObserverComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="targetNotesTab">
            <attribute name="title">
             <string>Notes</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QPlainTextEdit" name="targetNotesTextEdit"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="observationPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="KTitleWidget" name="observationTitle">
           <property name="text">
            <string>Observation Details</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="observationTabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="observationGeneralTab">
            <attribute name="title">
             <string>General</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="observationTargetLabel">
                 <property name="text">
                  <string>Target:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="observationObserverLabel">
                 <property name="text">
                  <string>Observer:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="observationObserverComboBox"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="observationSiteLAbel">
                 <property name="text">
                  <string>Site:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="observationSiteComboBox"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="observationSessionLabel">
                 <property name="text">
                  <string>Session:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="observationSessionComboBox"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="observationBeginLabel">
                 <property name="text">
                  <string>Begin:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="KDateTimeWidget" name="observationBeginDateTime"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="observationEndLabel">
                 <property name="text">
                  <string>End:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="KDateTimeWidget" name="observationEndDateTime"/>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="observationTargetComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Weather conditions</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="observationFaintestStarLabel">
                    <property name="text">
                     <string>Faintest star:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="observationFaintestStarSpinBox"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="observationSkyQualityLabel">
                    <property name="text">
                     <string>Sky quality:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QComboBox" name="observationSQUnitComboBox"/>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="observationSkyQualitySpinBox"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="observationSeeingLabel">
                    <property name="text">
                     <string>Seeing:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1" colspan="2">
                   <widget class="QComboBox" name="observationSeeingComboBox">
                    <item>
                     <property name="text">
                      <string>1 - Best conditions</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>2</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>3</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>4</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>5 - Worst conditions</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="observationEquipmentTab">
            <attribute name="title">
             <string>Equipment</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="observationScopeLabel">
                 <property name="text">
                  <string>Scope:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="observationScopeComboBox"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="observationEyepieceLabel">
                 <property name="text">
                  <string>Eyepiece:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="observationEyepieceComboBox"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="observationLensLabel">
                 <property name="text">
                  <string>Lens:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="observationLensComboBox"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="observationFilterLabel">
                 <property name="text">
                  <string>Filter:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="observationFilterComboBox"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="observationImagerLabel">
                 <property name="text">
                  <string>Imager:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="observationImagerComboBox"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="observationMagnificationLabel">
                 <property name="text">
                  <string>Magnification:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="observationMagnificationSpinBox"/>
               </item>
               <item row="6" column="1">
                <widget class="QPlainTextEdit" name="observationAccessoriesTextEdit"/>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="observationAccessoriesLabel">
                 <property name="text">
                  <string>Accessories:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="observationFindingsTab">
            <attribute name="title">
             <string>Findings</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="1">
                <widget class="QPlainTextEdit" name="observationResultsTextEdit"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="observationResultsLabel">
                 <property name="text">
                  <string>Results:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>KDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>kdatetimewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
